### ISSUE TITLE
Open_vsplit on variable in document_symbols causes exception

### ISSUE THREAD

**Issue by Gallactide** [nvim-neo-tree/neo-tree.nvim#989]
I'm trying to use the document_symbols source as a replacement for ctags+vista, it looks fantastic and I really want to keep using it, however when I hit open_vsplit on anything that doesn't have an expander icon next to it I get the following exception:

```
E5108: Error executing lua: ...d/neo-tree.nvim/lua/neo-tree/sources/common/commands.lua:574: Cursor position outside buffer                                                                                         
stack traceback:                                                                                                                                                                                                    
        [C]: in function 'nvim_win_set_cursor'                                                                                                                                                                      
        ...d/neo-tree.nvim/lua/neo-tree/sources/common/commands.lua:574: in function 'open'                                                                                                                         
        ...d/neo-tree.nvim/lua/neo-tree/sources/common/commands.lua:600: in function 'open_with_cmd'                                                                                                                
        ...d/neo-tree.nvim/lua/neo-tree/sources/common/commands.lua:625: in function 'func'                                                                                                                         
        .../nvim/plugged/neo-tree.nvim/lua/neo-tree/ui/renderer.lua:805: in function <.../nvim/plugged/neo-tree.nvim/lua/neo-tree/ui/renderer.lua:803>
```
After hitting return the document_symbols section is empty and a new empty split is opened.

My config is essentially the config from the readme with some icon changes:
```
  require("neo-tree").setup({
    nesting_rules = {},
    sources = {
      "filesystem",
      "buffers",
      "document_symbols",
      "git_status"
    },
    source_selector = {
      sources = {
        { source = "filesystem", display_name = " Files " },
        { source = "git_status", display_name = " Git " },
        { source = "document_symbols", display_name = " Symbols " },
      },
    },
    close_if_last_window = false, -- Close Neo-tree if it is the last window left in the tab
    enable_git_status = true,
    bind_to_cwd = true,
    enable_diagnostics = true,
    open_files_do_not_replace_types = { "terminal", "trouble", "qf" }, -- when opening files, do not use windows containing these filetypes or buftypes
    sort_case_insensitive = false, -- used when sorting files and directories in the tree
    default_component_configs = {
      modified = {
        symbol = "+",
        highlight = "Green",
      },
      container = {
        enable_character_fade = true
      },
      indent = {
        indent_size = 2,
        -- padding = 1, -- extra padding on left hand side
        -- indent guides
        with_markers = true,
        indent_marker = "│",
        last_indent_marker = "└", -- ╰
        highlight = "SpecialKey",
        -- expander config, needed for nesting files
        with_expanders = nil, -- if nil and file nesting is enabled, will enable expanders
        expander_collapsed = "",
        expander_expanded = "",
        expander_highlight = "NeoTreeExpander",
      },
      icon = {
        folder_closed = "",
        folder_open = "",
        folder_empty = "",
        --highlight = "Directory",
      },
      filesystem = {
        use_libuv_file_watcher = true,
        filtered_items = {
          visible = true, -- when true, they will just be displayed differently than normal items
          hide_dotfiles = false,
          hide_gitignored = false,
          hide_hidden = false,
          hide_by_name = {},
          hide_by_pattern = {},
          never_show = {},
          never_show_by_pattern = {},
        }
      }
    }
  })

```

**Comment by pysan3** [comment]
Duplicate of #898.

I'll ping people working over there to see if there were any progress ;)

**Comment by cseickel** [comment]
@Gallactide can you pull the latest from the `main` branch and test it out? I think this was fixed with #1005.
